### DISCUSSION TITLE
Little side project: AudiobookshelfApi - API SDK for audiobookshelf written in C#

### DISCUSSION THREAD

**Discussion by sandreas** [advplyr/audiobookshelf#2020] [Show and tell]
Hey @advplyr ,

just would like to show a new approach to use the audiobookshelf API:

https://github.com/sandreas/ToneAudioPlayer/tree/main/AudiobookshelfApi

I'll use this for my cross platform audio-player side project `ToneAudioPlayer`. It's pretty early, maybe someone else finds it useful.

Currently I'm looking for a way to autogenerate models and responses, probably I write a little script for this and adjust the issues manually.

I also plan to publish a nuget package as soon as it is in a releasable state. 

Feedback welcome.

sandreas

**Comment by advplyr** [comment]
Awesome, I've been working on the API a lot recently so the docs are a bit out of date just fyi.
